src/Mock.Chemistry/Mock.Chemistry.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.Quantum.Sdk/0.12.20070701-pull">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <IncludeQsharpCorePackages>false</IncludeQsharpCorePackages>    <!-- otherwise the standard library is included by the Sdk -->
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Quantum.Simulators" Version="0.12.20063023-beta" />
+  </ItemGroup>
+</Project>
```

Comment:
Any reason we don't want to choose an official release version here, e.g. `0.12.20070124`? (for both the `Sdk` and the `PackageReference`)

Reply:
It's just the version that iqsharp is currently using. We should upgrade the repo to the latest release if possible.